db/dcrpg/insightapi.go

```diff
@@ -155,16 +148,6 @@ func (pgb *ChainDB) GetBlockHash(idx int64) (string, error) {
 	return hash, nil
 }
 
-// AddressBalance returns a AddressBalance for the specified address,
-// transaction count limit, and transaction number offset.
-func (pgb *ChainDB) AddressBalance(address string, N, offset int64) (*dbtypes.AddressBalance, error) {
-	_, balance, err := pgb.AddressHistoryAll(address, N, offset)
```

Comment:
This was a hack to get the balance via AddressHistoryAll.  The N and offset args did nothing.